standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -610,6 +637,8 @@ struct Partition {
   10: optional i64 writeId=-1,
   11: optional bool isStatsCompliant,
   12: optional ColumnStatistics colStats // column statistics for partition
```

Comment:
Comma is missing after colStats. Does this still work fine, when you generate the code ?